modules/storages/app/common/storages/peripherals/storage_interaction/nextcloud/folder_files_file_ids_deep_query.rb

```diff
@@ -28,19 +28,44 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 
-module Storages::Peripherals::StorageInteraction::Nextcloud
-  class FolderFilesFileIdsDeepQuery
-    def self.call(storage:, folder:)
-      ::Storages::Peripherals::Registry
-        .resolve("nextcloud.queries.propfind")
-        .call(
-          storage:,
-          depth: "infinity",
-          path: folder.path,
-          # nc:acl-list is only required to avoid https://community.openproject.org/wp/49628. See comment #4.
-          props: %w[oc:fileid nc:acl-list]
-        ).map do |obj|
-        obj.transform_values { |value| Storages::StorageFileInfo.from_id(value["fileid"]) }
+module Storages
+  module Peripherals
+    module StorageInteraction
+      module Nextcloud
+        class FolderFilesFileIdsDeepQuery
```

Comment:
🟢 The name of query is... weird, but I don't have a good suggestion to what to change it to. It returns a dictionary of full remote file path and ids, and operates recursively given a start point.

The recursive part is kind of a detail since we don't specify a maximum depth. `FilePathToIdMapQuery`? `WeirdThingWeNeedQuery`?

Reply:
`FilePathToIdMapQuery` doesn't sound too weird. Should I just issue a small renaming PR after this one?

Reply:
Sure, why not.